synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -346,16 +411,29 @@ export class SynopsysToolsParameter {
     }
 
     if (parseToBoolean(inputs.COVERITY_AUTOMATION_PRCOMMENT)) {
-      console.info("Coverity Automation comment is enabled");
-      covData.data.azure = await this.getAzureRepoInfo();
-      covData.data.environment = this.setEnvironmentScanPullData();
-      covData.data.coverity.automation.prcomment = true;
+      if (!isPullRequest()) {
```

Comment:
can we store the isPullRequest() value in some variable and reuse here? we have used earlier in case of stream name as well.

Reply:
updated